### DISCUSSION TITLE
Nested Layer, Options for Sublayers?

### DISCUSSION THREAD

**Discussion by randomqname** [nftchef/art-engine#134] [Help + Q&A] [answered]
Essentially I'm trying to make 2 png's (hair +eyebrows) generate togethe; for example the main folder is for Hair where one set of hair is chosen, but then depending on the hair chosen a set of eyebrows are nested and then one is randomly selected. Would it be possible to configure something like this? Not sure if my syntax is appropriate but any advice would be appreciated!

|--Hair
|---- Hair Type 1.png
|----  Hair Type 1 EyeBrow Option 1.png
|----  Hair Type 1 EyeBrow Option 2.png
|----  Hair Type 1 EyeBrow Option 3.png

**Comment by randomqname** [comment, marked as answer]
Update managed to figure out how to do it via nested structures breakdown below for anyone that might be in a similar boat

(main folder) Hair

--(sub folder) Special Hair 1#10
---(nested in subfolder) Special Hair 1.png
---(nested in subfolder) Eyebrow A for Special Hair1#10.png
---(nested in subfolder) Eyebrow B for Special Hair1#10.png

--(sub folder 2) Special Hair 2#10
---(nested in subfolder2) Special Hair 2.png
---(nested in subfolder2) Eyebrow A for Special Hair1#10.png
---(nested in subfolder2) Eyebrow B for Special Hair1#10.png

etc etc.

Essentially syntax is very important; you want to make sure you add a '#' weight to anything that's optional and then leave anything mandatory with no '#' so it's always selected. In this case the Special Hair.png is mandatory and then the eyebrow.png for that specific hair has weights which then randomly selects one!

**Reply by cryptofannatic** [reply]
did you do this in layer configurations or via forced combinations?
